lib/modules/onc_program/onc_standalone_public_launch_sequence.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+require_relative './shared_onc_launch_tests'
+
+module Inferno
+  module Sequence
+    class OncStandalonePublicLaunchSequence < SequenceBase
+      include Inferno::Sequence::SharedONCLaunchTests
+
+      title 'Public Client Standalone Launch Sequence'
+
+      description 'Register Inferno as a public client with patient access and execute standalone launch.'
+
+      test_id_prefix 'OSLSP'
+
+      requires :onc_public_client_id,
+               :onc_public_scopes,
+               :onc_sl_oauth_authorize_endpoint,
+               :onc_sl_oauth_token_endpoint,
+               :initiate_login_uri,
+               :redirect_uris
+
+      defines :token, :id_token, :refresh_token, :patient_id
+
+      show_uris
+
+      def valid_resource_types
+        [
+          '*',
+          'Patient',
+          'AllergyIntolerance',
+          'CarePlan',
+          'CareTeam',
+          'Condition',
+          'Device',
+          'DiagnosticReport',
+          'DocumentReference',
+          'Encounter',
+          'Goal',
+          'Immunization',
+          'Location',
+          'Medication',
+          'MedicationOrder',
+          'MedicationRequest',
+          'MedicationStatement',
+          'Observation',
+          'Organization',
+          'Practitioner',
+          'PractitionerRole',
+          'Procedure',
+          'Provenance'
+        ]
+      end
+
+      def required_scopes
```

Comment:
This doesn't appear to be ever used.  `onc_standalone_launch_tests` use this with `required_scope_test` from the `shared_onc_launch_tests`.  I'm not sure you'd want to use that test out of the box though because the title says "access with OpenID Connect and Refresh Token scopes used" and those scopes aren't actually checked.